hazelcast/src/main/java/com/hazelcast/sql/impl/schema/Table.java

```diff
@@ -28,20 +28,23 @@
 public abstract class Table {
 
     private final String schemaName;
-    private final String name;
+    private final String sqlName;
+    private final String objectName;
```

Comment:
I am not sure that `objectName` should be in the `Table` class. Having the separate name for the underlying object, and even the presence of this underlying object per se is an implementation detail.
It feels like the `AbstractMapTable` class is a better candidate for this field.

Reply:
Every object in IMDG does have it (imap and replicated map). Also every object in Jet does have, though it could have different names (like mapName, topicName etc). I moved it up here so the distinction is more clear.

Reply:
I still do not get it. This class should contain fields that is common for all tables. `objectName` is not a common concept across all possible object types that we may have in future. E.g. it is not applicable to a table backed by a file. Also it is not applicable to views.

I think, `objectName` should belong to `AbstractMapTable`.

Reply:
¯\_(ツ)_/¯